app/models/concerns/standardizable.rb

```diff
@@ -54,22 +54,26 @@ def override_setters
         when 'facility_code' then override_facility_code_setter
         when 'institution' then override_institution_setter
         when 'state' then override_state_setter
+        when 'ope' then override_ope_setter
+        when 'cross' then override_cross_setter
         else override_generic_setter(col, data_type)
         end
       end
     end
 
     def override_facility_code_setter
+      klass = self
+
       define_method 'facility_code=' do |value|
-        value = value.try(:strip)
-        self['facility_code'] = value ? value.upcase.rjust(8, '0') : nil
+        value = value.try(:strip).try(:upcase)
+        self['facility_code'] = klass.forbidden_word?(value) || value.blank? ? nil : value.rjust(8, '0')
       end
     end
 
     def override_institution_setter
       define_method 'institution=' do |value|
-        value = value.try(:strip)
-        self['institution'] = value ? value.upcase : nil
+        value = value.try(:strip).try(:upcase)
+        self['institution'] = value ? value : nil
```

Comment:
here as well, dynamically defining a method using meta programming doesn't seem necessary, perhaps I'm missing context, but this is just confusing.